geomesa-cassandra/geomesa-cassandra-tools/bin/geomesa-cassandra

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+# Set environment variables in bin/geomesa-env.sh
+
+if [ -z "${GEOMESA_CASSANDRA_HOME}" ]; then
```

Comment:
We've been replacing the direct home names with references filled in by the -dist pom. So I'd replace all references to GEOMESA_CASSANDRA_HOME with %%gmtools.dist.name%%_HOME and add 
`<properties>`
`    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>`
`    <gmtools.dist.name>GEOMESA_CASSANDRA</gmtools.dist.name> <!-- used in tools filtering -->`
`</properties>`
to geomesa-cassandra-dist/pom.xml